build.xml

```diff
@@ -351,58 +351,6 @@
     <fail message="Test failure detected, check test results." if="test.failed" />
   </target>
 
-  <target name="test_quick_no_handle_coverage"
```

Comment:
This was useful when the handle service was a nightmare to deploy, but now it's pretty easy and it's not used anywhere